src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1970,59 +1970,87 @@ limitations under the License.
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
```

Comment:
Why is Identify server still part of this device type?

Reply:
Removed.